source/developer-guides/build-system.rst

```diff
@@ -33,6 +33,25 @@ ConfigDataBlob.h             C file for *Internal* configuration data with defau
 See :ref:`customize-build` for more details.
 
 
+.. _sbl-keys:

```

Comment:
Is this the right place for this info? I think build-host-setup.rst maybe better?

This section talks about the EDK2 build infrastructure.

Reply:
Also, add a reference to this section here https://github.com/slimbootloader/documentation/pull/74/files#diff-7878704aff6f8f291ef221a31c437735

Reply:
Updated in build-host-setup section